src/Mono.Android/Android.App/SyncContext.cs

```diff
@@ -14,7 +15,15 @@ public override SynchronizationContext CreateCopy ()
 
 		public override void Post (SendOrPostCallback d, object state)
 		{
-			using (Handler h = new Handler(Application.Context.MainLooper)) {
+			var looper = Application.Context?.MainLooper;
+			if (looper == null) {
+				var message = "No Looper available to this SynchronizationContext, callback will not be executed.";
```

Comment:
"Warning: No Android message loop is available. Skipping invocation of `{d.Method.DeclaringType.FullName}.{d.Method.Name}`!"

This reduces terms like "Looper" ("wtf is a looper!"), replacing with a (hopefully?) more common term of "message loop", and also provides the callback method that won't be invoked. This should hopefully be useful for debugging/diagnostics.